DPackRx/Features/FileBrowser/FileBrowserControl.xaml

```diff
@@ -176,7 +176,7 @@
                         </GridViewColumn.CellTemplate>
                     </GridViewColumn>
                     <!-- File name -->
-                    <GridViewColumn DisplayMemberBinding="{Binding FileName, Mode=OneTime}" Width="200">
+                    <GridViewColumn DisplayMemberBinding="{Binding FileNameWithPath, Mode=OneTime}" Width="200">
```

Comment:
Don't think I want for it to work that way tbhwy. I think old pre-VS2022 version used to show project relative path. That would be of some value. Otherwise you gonna see nothing but start of the file path there.

Reply:
That’s fine, to be honest, I just made this pr without testing it at all. Sorry about that. Just wanted to demonstrate roughly what I want.